fastly/block_fastly_service_v1_package.go

```diff
@@ -0,0 +1,107 @@
+package fastly
+
+import (
+	"fmt"
+	"log"
+
+	gofastly "github.com/fastly/go-fastly/fastly"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+type PackageServiceAttributeHandler struct {
+	*DefaultServiceAttributeHandler
+}
+
+func NewServicePackage() ServiceAttributeDefinition {
+	return &PackageServiceAttributeHandler{
+		&DefaultServiceAttributeHandler{
+			key: "package",
+		},
+	}
+}
+
+func (h *PackageServiceAttributeHandler) Register(s *schema.Resource) error {
+	s.Schema[h.GetKey()] = &schema.Schema{
+		Type:     schema.TypeList,
+		Required: true,
+		MaxItems: 1,
+		MinItems: 1,
+		Elem: &schema.Resource{
```

Comment:
Since this doesn't have any required fields, what happens when you just have an empty `package` resource block? Does this actually create something on the backend?

Reply:
Filename is now required